addons/website_event_booth/views/event_booth_templates.xml

```diff
@@ -22,22 +22,38 @@
                         </div>
                     </div>
                 </section>
-                <div t-if="event.is_finished" class="container d-flex flex-column flex-grow-1">
-                    <span class="alert alert-info">This event is finished. It's no longer possible to book a booth.</span>
+                <div t-if="event.is_finished">
+                    <div class="col-12">
+                        <div class="d-flex flex-column align-items-center justify-content-center text-center">
```

Comment:
These flex classes shouldn't be necessary... 
The first `<div>` will take an entire row, the following `<h2>` will take another one (both are `display: block` by default).
 could you check?